specification/synapse/resource-manager/Microsoft.Synapse/preview/2020-04-01-preview/sqlDatabase.json

```diff
@@ -0,0 +1,299 @@
+          },
+          "202": {
+            "description": "Updating the database is in progress."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Updates a sqldatabase.": {
+            "$ref": "./examples/PatchSqlDatabase.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SqlDatabaseProperties": {
```

Comment:
if these are created asynchronously they should have a "provisioningState" property in their property bag.  See the async PUT link I pasted above